Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(platforms: ['windows-i686', 'windows-x86_64'])
+buildPlugin()
```

Comment:
@rechi you mean this?

Reply:
that means it will build for all platforms.
If you don't want that add 'ubuntu-ppa' to the list

Reply:
Yes, missed that as commit before `Add debian build package` was green and because of that looked only into Jenkinsfile of base branch.

Does it (now) also support OpenGLES?

Reply:
Not direct possible to test GLES, but it should (I will try it at some point).

Reply:
android-aarch64 works

Reply:
Cool, don't merge! Have added wrongly some c&p files (settings.xml) who not from them ;-)